SDAPackage/SDA/obstacle_map.py

```diff
@@ -216,49 +221,18 @@ def does_path_intersect_obstacle_2d(self, obstacle, uav_point, waypoint):
 
         # Uncomment for DEBUGGING ONLY
-        print("Waypoint Vector: " + str(waypoint_vector))
-        print("Obstacle Vector: " + str(obstacle_vector))
-        print("Rejection Vector: " + str(rejection_vector))
-        print("Rejection Vector Magnitude: " + str(rejection_vector_magnitude))
-        print("Obstacle Radius: " + str(obstacle.get_radius()))
-        print("Distance From Obstacle: " + str(VectorMath.get_vector_magnitude(np.subtract(uav_point, obstacle.get_point()))))
+        #print("Waypoint Vector: " + str(waypoint_vector))
+        #print("Obstacle Vector: " + str(obstacle_vector))
+        #print("Rejection Vector: " + str(rejection_vector))
+        #print("Rejection Vector Magnitude: " + str(rejection_vector_magnitude))
+        #print("Obstacle Radius: " + str(obstacle.get_radius()))
+        #print("Distance From Obstacle: " + str(VectorMath.get_vector_magnitude(np.subtract(uav_point, obstacle.get_point()[:2]))))
 
         if self.is_obstacle_in_path_of_drone(obstacle_vector, waypoint_vector):
             return rejection_vector_magnitude < obstacle.get_radius()
 
         return False
 
-    def does_path_intersect_obstacle_3d(self, obstacle, drone_point, waypoint):
-        """
-        Determine if the vector between a UAV's position and the current waypoint intersect
-        an obstacle.
-
-        :param obstacle: The obstacle to determine if the UAV intersects with
-        :type obstacle: StationaryObstacle or MovingObstacle
-        :param uav_point: The UAV's position
-        :type uav_point: Numpy Array
-        :param waypoint: The waypoint that the UAV is headed to
-        :type waypoint: Numpy Array
-        """
-        waypoint_vector = np.subtract(waypoint, drone_point)
-        obstacle_vector = np.subtract(obstacle.get_point(), drone_point)
-        obstacle_vector_magnitude = VectorMath.get_vector_magnitude(obstacle_vector)
-        rejection_vector = VectorMath.get_vector_rejection(obstacle_vector, waypoint_vector)
-        rejection_vector_magnitude = VectorMath.get_vector_magnitude(rejection_vector)
-
-        # Uncomment for DEBUGGING ONLY
-        print("Waypoint Vector: " + str(waypoint_vector))
-        print("Obstacle Vector: " + str(obstacle_vector))
-        print("Rejection Vector: " + str(rejection_vector))
-        print("Rejection Vector Magnitude: " + str(rejection_vector_magnitude))
-        print("Obstacle Radius: " + str(obstacle.get_radius()))
-        print("Distance From Obstacle: " + str(VectorMath.get_vector_magnitude(np.subtract(drone_point, obstacle.get_point()))))
-
-        if self.is_obstacle_in_path_of_drone(obstacle_vector, waypoint_vector):
-            return rejection_vector_magnitude < Constants.STATIONARY_OBSTACLE_SAFETY_RADIUS
-
-        return False
-
     def is_obstacle_in_path_of_drone(self, obstacle_vector, waypoint_vector):
         """
         Looks at the signs of the components of the vectors to determine if the
```

Comment:
All function docs should define args, type of args, and what it returns (if it returns anything.)

Example:

```
def is_banana(self, fruit):
    """
    Determines whether or not a fruit object is a banana.

    :param fruit:    Object to be evaluated.
    :type fruit:     Fruit object.

    :return:         True/false.
    """
```

See additional example in @Inffzy 's code: https://github.com/FlintHill/SUAS-Competition/blob/master/UpdatedImageProcessing/TargetDetection/automatic_tester.py#L10-L25

Reply:
@BrightStraightReality you do this in some places, but not everywhere.